supabase/functions/process-document/index.ts

```
@@ -116,18 +286,40 @@ serve(async (req) => {
+      case 'ocr_extraction':
+        // Queue for OCR processing if needed
+        const { data: queueItem } = await supabaseClient
+          .from('pdf_processing_queue')
+          .insert({
+            pdf_file_id: pdfFileId,
+            priority,
+            processing_type: 'ocr',
+            processing_options: { language }
+          })
+          .select()
+          .single()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)